src/components/ebay-notice/examples/4-dismissible-notice/template.marko

```diff
@@ -0,0 +1,7 @@
+<ebay-notice
+    status="information"
+    type="page"
+    aria-text="information" dismissible=true aria-close="Dismiss notice">
```

Comment:
Minor, but this can just be `dismissable` alone, instead of `dismissable=true`, right?

Reply:
Yes I missed this, will change it